package.json

```diff
@@ -381,7 +381,7 @@
     "ts-loader": "^7.0.5",
     "ts-node": "^8.10.2",
     "type-fest": "^0.18.0",
-    "typedoc": "^0.19.2",
+    "typedoc": "0.17.0-3",
```

Comment:
Why the downgrade?

Reply:
Library mode is only supported in this specific version. I'm still investigating if we can get similar results with newer versions.

Reply:
Fair (weird) but fair.